cuml/metrics_demo.ipynb

```diff
@@ -43,7 +43,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "n_samples = 2**10\n",
+    "n_samples = 2**5\n",
```

Comment:
This is a really low value. Is there a reason for the change? Even 2^10 is pretty low.

Reply:
oops changed it while testing